api/src/main/java/com/nicknathanjustin/streamercontracts/alerts/AlertServiceImpl.java

```diff
@@ -1,17 +1,44 @@
 package com.nicknathanjustin.streamercontracts.alerts;
 
+import com.google.common.hash.Hashing;
+import com.nicknathanjustin.streamercontracts.contracts.dtos.Contract;
+import com.nicknathanjustin.streamercontracts.users.UserModel;
 import lombok.NonNull;
 import lombok.RequiredArgsConstructor;
 import org.springframework.messaging.simp.SimpMessageSendingOperations;
 
+import java.nio.charset.StandardCharsets;
+
 @RequiredArgsConstructor
 public class AlertServiceImpl implements AlertService {
 
     @NonNull final private SimpMessageSendingOperations messagingTemplate;
 
     @Override
-    public void sendAlert(@NonNull final String alertChannelId, @NonNull final String title, @NonNull final String description) {
-        final AlertMessage message = new AlertMessage(title, description);
+    public void sendNotification(@NonNull final UserModel user, @NonNull final String title, @NonNull final String description) {
+        final NotificationMessage message = new NotificationMessage(title, description);
+        final String url = "/alert/" + user.getTwitchUsername();
+
+        messagingTemplate.convertAndSend(url, message);
+    }
+
+    @Override
+    public void sendStreamActivateAlert(@NonNull final UserModel user, Contract contract) {
```

Comment:
This method and the following are duplicated minus the AlertType... You could get rid of these 2 methods and make a new one that takes in an alertType param.

```
@Override
public void alertStream(@NonNull final UserModel user, @NonNull final Contract contract, @NonNull final AlertType alertType) {
//code
}
```